docs/installation/est/configuring-est-fullcmc-example.adoc

```diff
@@ -0,0 +1,977 @@
+pki_http_port=8080
+pki_server_database_password=Secret.123
+
+[Tomcat]
+pki_ajp_port=8009
+pki_tomcat_server_port=8005
+
+[CA]
+pki_admin_email=caadmin@example.com
+pki_admin_name=caadmin
+pki_admin_nickname=caadmin
+pki_admin_password=Secret.123
+pki_admin_uid=caadmin
+
+pki_client_pkcs12_password=Secret.123
+
+pki_ds_url=ldap://pki.example.com:2389
+pki_ds_base_dn=dc=ca,dc=pki,dc=example,dc=com
+pki_ds_database=ca
+pki_ds_password=Secret.123
+
+pki_security_domain_name=EXAMPLE
+
+pki_ca_signing_nickname=ca_signing
+pki_ocsp_signing_nickname=ca_ocsp_signing
+pki_audit_signing_nickname=ca_audit_signing
+pki_sslserver_nickname=sslserver
+pki_subsystem_nickname=subsystem
+----
+
+=== Step 8: Install CA
+
+[source,bash]
+----
+pkispawn -f /root/dogtag/pki-ca/ca.cfg -s CA
+----
+
+The CA admin certificate will be exported to `/root/.dogtag/pki-ca/ca_admin_cert.p12`.
+
+=== Step 9: Setup CA Admin User
+
+After installation, the admin certificate and key are stored in `~/.dogtag/pki-ca/ca_admin_cert.p12`.
+
+To use the admin certificate, do the following:
+
+. As root, export the CA signing certificate from the server NSS database and copy it to the working directory:
++
+[source,bash]
+----
+# pki-server cert-export ca_signing --cert-file /tmp/ca_signing.crt -i pki-ca
+# chmod 644 /tmp/ca_signing.crt
+# cp /tmp/ca_signing.crt /root/dogtag/pki-ca/
+----
+
+. As regular user, copy the CA signing certificate to your working directory and import it into the client NSS database:
++
+[source,bash]
+----
+cp /tmp/ca_signing.crt ~/.dogtag/pki-ca/
+pki nss-cert-import --cert ~/.dogtag/pki-ca/ca_signing.crt --trust CT,C,C ca_signing
+----
+
+. Import the admin certificate and key into the client NSS database (by default ~/.dogtag/nssdb):
++
+[source,bash]
+----
+pki pkcs12-import \
+    --pkcs12 ~/.dogtag/pki-ca/ca_admin_cert.p12 \
+    --password Secret.123
+----
+
+. Verify that the admin certificate can be used to access the CA subsystem:
++
+[source,bash]
+----
+pki -n caadmin ca-user-show caadmin
+----
++
+NOTE: The `pki` command uses `~/.dogtag/nssdb` as the default NSS database directory. If you need to use a different NSS database location, specify it with the `-d` parameter.
+
+=== Step 10: Configure CA for EST CMC Responses
+
+IMPORTANT: The `cmc.response.useSimpleOnSuccess` parameter changes CMC response behavior for **all** CMC requests handled by this CA, not just EST. Setting it to `true` makes the CA return simple (non-full) CMC responses on successful enrollment, which is required for EST RFC compliance but may affect other CMC clients.
+
+As root, edit `/var/lib/pki/pki-ca/ca/conf/CS.cfg` and add:
+
+[source,properties]
+----
+cmc.response.useSimpleOnSuccess=true
+----
+
+Restart the CA:
+
+[source,bash]
+----
+pki-server restart pki-ca
+----
+
+NOTE: This setting is automatically added by the upgrade script for existing CA installations. For new installations, it must be manually configured if EST fullcmc support is needed.
```

Comment:
If new instances require to add it  manually, it should not be modified in existing instance from upgrade script!